docs/montecarlo/sourceintegration.rst

```diff
@@ -0,0 +1,102 @@
+
+After the final monte carlo step, a level absorption estimator is calculated, given by:
+
+.. math::
+
+   \dot E_u = \sum_{i < u} \dot E_{lu}
+
+that is, by summing all the line absorption estimators below the curently selected level. In the code this is done with a bit of pandas magic in `make_source_function` found in `simulation/base.py`. By creating a dataframe using the estimated :math:`\dot E_u` and appyling to it a copy of the index of the atomic data, the sum above can be done with to a `groupby` operation following by a sum of the result. 
+
+The source function for each line can then be derived from the relation
+
+.. math::
+
+   \left( 1- e^{-\tau_lu}\right) S_{ul} = \frac{\lambda_{ul} t}{4 \pi} q_{ul} \dot E_u
+
+where :math:`\lambda_{ul}` is the wavelength of each line  :math:`u \rightarrow l`, and :math:`q_{ul}` is the corresponding branching ratio. The attenuating factor is kept on the left hand side because it is the product of the two that will appear in later formulae. The product on the right hand side is also evaluated in `make_source_function`. 
+
+Having thus produced attenuated source functions from our Monte Carlo run, we move on to using this to calculate the emerging intensity and finally the luminosity per wavelength. The final integral is given as 
+
+.. math::
+
+   L_\nu  = 8 \pi^2 \int_0^\infty I_\nu (p) p dp
+
+where :math:`p` is the impact parameter of a ray trough the supernova envelope that reaches the distant observer, and :math:`I_\nu (p)` is the intensity along one such ray, given by recursing trough the list of attenuated source functions from the blue to the red and adding up contributions. The relation linking the intensity before the k:th transition :math:`u \rightarrow l` to the intensity after is 
+
+.. math::
+
+   I_k^r = I_k^b e^{-\tau_k} + \left( 1- e^{-\tau_k}\right) S_{k}
+
+where the superscripts are crucial, with :math:`r` and :math:`b` referencing the red and blue sides of the k:th transition respectively. To go from the red side of a line to the blue side of the next we can either ignore continuum sources of opacity, in which case
+
+.. math:: 
+
+   I_{k+1}^b = I_k^r
+
+or include them, then requiring we perform
+
+.. math:: 
+
+   I_{k+1}^b = I_k^r + \Delta \tau_k \left[ \frac 1 2(J_k^r + J_{k+1}^b) - I_k^r  \right]
+
+The starting condition for the blue to red side transition is either :math:`I_0^r = 0` for the case that the impact parameter is greater than the radius if the photosphere, or :math:`I_0^r = B_\nu(T)` if the impact parameter is less than the radius of the photosphere. 
+
```

Comment:
Shouldn't this be I_0^b (if index 0 is the first line in the list)? Make T s--> T_ph and refer to it as the temperature at the photosphere. Would also be good to additionally define the two regimes as p>R_ph and p<R_ph.